lms/static/scripts/frontend_apps/config.js

```diff
@@ -27,9 +27,10 @@ import { createContext } from 'preact';
 /**
  * @typedef StudentInfo
  * @prop {string} displayName
- * @prop {string} userid
+ * @prop {string} userid - Identifier for student user in `h`
  * @prop {string} LISResultSourcedId - Unique outcome identifier
  * @prop {string} LISOutcomeServiceUrl - API URL for posting outcome results
+ * @prop {string} lmsId - Identifier for student user in the LMS
```

Comment:
Is this the LMS-determined identifier or something generated internally by our LMS app?